GOVERNANCE.md

```diff
@@ -1,30 +1,46 @@
+  for incoming PRs (i.e., all maintainers are expected to shoulder a proportional share of
   community reviews).
 * After a period of approximately 2-3 months of working together and making sure we see eye to eye,
-  the existing senior committers will confer and decide whether to grant commit status or not. We
-  make no guarantees on the length of time this will take, but 2-3 months is the approximate
+  the existing senior maintainers will confer and decide whether to grant maintainer status or not.
+  We make no guarantees on the length of time this will take, but 2-3 months is the approximate
   goal.
 
-## Your organization is currently a committer
+## Your organization is currently a maintainer
 
-* First decide whether your organization really needs more people with commit access. Valid reasons
-  are "blast radius", a large organization that is working on multiple unrelated projects, etc.
-* Contact a senior committer for your organization and express interest.
-* Start doing PRs and code reviews under the guidance of your senior committer.
-* After a period of 1-2 months the existing senior committers will discuss granting "standard"
-  commit access.
-* "Standard" commit access can be upgraded to "senior" commit access after another 1-2 months of
-  work and another conference of the existing senior committers.
+* First decide whether your organization really needs more people with maintainer access. Valid
+  reasons are "blast radius", a large organization that is working on multiple unrelated projects,
+  etc.
+* Contact a senior maintainer for your organization and express interest.
+* Start doing PRs and code reviews under the guidance of your senior maintainer.
+* After a period of 1-2 months the existing senior maintainers will discuss granting "standard"
+  maintainer access.
+* "Standard" maintainer access can be upgraded to "senior" maintainer access after another 1-2
+  months of work and another conference of the existing senior maintainers.
+
+# Conflict resolution and voting
+
+In general, we prefer that technical issues and maintainer membership are amicably worked out
+between the persons involved. If a dispute cannot be decided independently, the maintainers can be
+called in to decide an issue. If the maintainers themselves cannot decide an issue, the issue will
+be resolved by voting. The voting process is a simple majority in which each senior maintainer
+receives two votes and each normal maintainer receives one vote.
+
+# Adding new projects to the envoyproxy GitHub organization
+
+New projects will be added to the envoyproxy organization via GitHub issue discussion in one of the
+existing projects in the organization. Once sufficient discussion has taken place (~3-5 business
+days but depending on the volume of conversation), the maintainers will decide whether the new
+project should be added (see the section above on voting if the maintainers cannot easily decide).
```

Comment:
Are maintainers on the Envoy project maintainers with same seniority across all projects in envoyproxy? I think (?) that is what's being done today, but for example, I'm providing little value on `go-control-plane` and it might make sense as we scale to have project specific maintainers who can provide leadership in the areas of their focus and expertise.

Reply:
Agreed, let me try to clarify this.